tests_integration/nativeapp/test_events.py

```diff
@@ -27,7 +27,7 @@
 
 
 @pytest.mark.integration
-def test_app_events(runner, temporary_working_directory):
+def test_app_events_no_event_table(runner, temporary_working_directory):
```

Comment:
I could add a test that creates a new event table, raw inserts events into it, and reads them back, but I'm worried about it being slow and/or flaky. Thoughts?